client/src/components/Donation/donation-modal-body.tsx

```diff
@@ -0,0 +1,306 @@
+            <b>
+              {t('donate.nicely-done', {
+                block: t(
+                  `intro:${recentlyClaimedBlock.superBlock}.blocks.${recentlyClaimedBlock.block}.title`
+                )
+              })}
+            </b>
+          )}
+          <h2>{t('donate.help-us-develop')}</h2>
+        </Col>
+      </Row>
+    );
+  } else if (!showForm) {
+    return (
+      <Row className='text-center block-modal-text'>
+        <Col sm={10} smOffset={1} xs={12}>
+          <h2>{t('donate.modal-benefits-title')}</h2>
+        </Col>
+      </Row>
+    );
+  } else {
+    return null;
+  }
+}
+
+function CloseButtonRow({
+  donationAttempted,
+  closeDonationModal
+}: {
+  donationAttempted: boolean;
+  closeDonationModal: () => void;
+}) {
+  const { t } = useTranslation();
+  return (
+    <Row>
+      <Col sm={4} smOffset={4} xs={8} xsOffset={2}>
+        <button
+          className='close-button'
+          type='button'
+          onClick={closeDonationModal}
+        >
+          {donationAttempted ? t('buttons.close') : t('buttons.ask-later')}
+        </button>
+      </Col>
+    </Row>
+  );
+}
+
+const Benefits = ({
+  setShowForm,
+  executeGA
+}: {
+  setShowForm: (arg: boolean) => void;
+  executeGA: (arg: { event: string; action: string }) => void;
+}) => {
+  const { t } = useTranslation();
+
+  const handleBecomeSupporterClick = () => {
+    executeGA({
+      event: 'donation_related',
+      action: `Modal Become Supporter Click`
+    });
+    setShowForm(true);
+  };
+  return (
+    <Row className={'donate-btn-group'}>
+      <Col xs={12}>
+        <ModalBenefitList />
+        <Spacer size='small' />
+        <button
+          className='text-center confirm-donation-btn donate-btn-group'
+          type='submit'
+          onClick={handleBecomeSupporterClick}
+        >
+          {t('donate.become-supporter')}
+        </button>
+        <Spacer size='medium' />
+      </Col>
+    </Row>
+  );
+};
+
+const AnimationContainer = ({
+  setIsAnimationVisible
+}: {
+  setIsAnimationVisible: (arg: boolean) => void;
+}) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <p style={{ opacity: 0, position: 'absolute' }}>
+        {t('donate.animation-description')}{' '}
+      </p>
+      <div className='donation-animatoin-container' aria-hidden='true'>
+        <div className='donation-animation-bullet-points'>
+          <p className='donation-animation-bullet-1'>
+            {t('donate.become-supporter')}
+          </p>
+          <p className='donation-animation-bullet-2'>
+            {t('donate.remove-distractions')}
+          </p>
+          <p className='donation-animation-bullet-3'>
+            {t('donate.reach-goals-faster')}
+          </p>
+          <p className='donation-animation-bullet-4'>
+            {t('donate.help-millions-learn')}
+          </p>
+        </div>
+        <img
+          alt={'asdfasd'}
+          src={donationAnimation}
+          id={'donaition-animation'}
```

Comment:
```suggestion
          id={'donation-animation'}
```
Other half of the typo